### DISCUSSION TITLE
Configure icon theme used by systray?

### DISCUSSION THREAD

**Discussion by the-argus** [qtile/qtile#3372] [Q&A] [answered]
As far as I can tell, there is no way to configure the icon theme used by the system tray. I know I'm missing something, because I somehow accidentally did it once before, but I cannot for the life of me figure out what I did.
I know the systray is jank, I think the icons are drawn by the applications and not the WM, right? If so then there's probably some way to do this involving changing index.theme files and .desktop files, but I haven't been able to find it and would like to be pointed in the right direction.

**Comment by m-col** [comment, marked as answer]
> I think the icons are drawn by the applications and not the WM, right?

Yep - this is the case. I think it often depends on what toolkit the application uses. E.g. GTK apps will use GTK icon sets for theirs, I think. So for those it would be some kind of GTK settings. Presumably there is something equivalent for Qt. I found I can change these with the `lxappearance` and `qt5ct` utilities respectively - they are in the Arch repos, likely others too.

Other programs like VLC will have theirs set by themselves, so I think the way to change that would be by editing the desktop file.

Hope that helps.

**Reply by elParaguayo** [reply]
For GTK apps, I *think* you need to edit `~/.gtkrc-2.0` and `~/.config/gtk-{3,4}.0/settings.ini`. Not tried it though.

**Reply by the-argus** [reply]
I had actually tried all of these things, and none of them worked, which is why I came here, thinking that maybe the WM had something to do with it. I changed the gtk settings for 2.0 and 3.0, as well as my default index.theme inherits, but it didn't work until restart. I tried reverting them and restarting one-by-one to figure out which one did it, but even after removing all the changes I made, the icon set stayed fixed. It remains a mystery!
